updates

magrefaat opened the pull request.

mshaban-msft commented:
@magrefaat 
115 files changed!
can you please provide a change log of what did you do?

magrefaat commented:
@mshaban-msft updated redame file and indexing pipeline illustration. moved the cutom text sample function and configs file to folder root, renamd olution to source code and removed the unneeded schema file
